containers/init/ready/ready.go

```diff
@@ -89,6 +98,20 @@ type LoadTestGetter interface {
 	Get(context.Context, string, metav1.GetOptions) (*grpcv1.LoadTest, error)
 }
 
+// NodeInfo contain pod name, pod IP and node name in which the pod reside for one worker or driver.
+type NodeInfo struct {
+	Name     string
+	PodIP    string
+	NodeName string
+}
+
+// NodesInfo contain all pods' NodeInfo included in a load test.
```

Comment:
Perhaps "NodesInfo contains NodeInfo for all pods included in a load test."

Reply:
Done, thanks!